runtime/src/ancient_append_vecs.rs

```diff
@@ -3042,6 +3053,11 @@ pub mod tests {
 
             combine_ancient_slots_packed_for_tests(&db, (next_slot..=max_slot_inclusive).collect());
 
+            assert_eq!(
+                crate::append_vec::APPEND_VEC_MMAPPED_FILES_OPEN.load(Ordering::Relaxed),
+                2 + MAX_RECYCLE_STORES as u64
```

Comment:
Thanks for the comment above.  So this would reflect the total number of append-vecs that have been ever created during the test.

One related question:  so what is the current number of append-vecs + ancient-append-vec?  is it one ancient-append-vec or two?

Reply:
not ever created. it reflects how many are STILL alive.
this is all held by recycling + any ancient append vecs + any non-ancient append vecs.
so, squashing 2 append vecs into 1 ancient append vec will result in 1 alive append vec and conceptually the original 2 append vecs deleted. Except, recycling holds onto both old ones, so we end up with 3 alive in this example.

Reply:
Thanks for the detailed answer!